cmd/_integration-tests/Makefile

```diff
@@ -7,20 +7,28 @@ TRANSPORT_TEST_MSG=\n$(OK_COLOR)Starting transport end to end test:$(END_COLOR_L
 
 CLI_TEST_MSG=\n$(OK_COLOR)Start server and cliclient generate, build, and run test:$(END_COLOR_LINE)
 
+export PATH := $(CURDIR):$(PATH)
 
 all: test
 
 test: clean test-transport test-cli
 
-test-transport:
+local-tools:
+	go build -o protoc-gen-truss-protocast github.com/TuneLab/go-truss/cmd/protoc-gen-truss-protocast
+	go build -o truss github.com/TuneLab/go-truss/cmd/truss
+
+test-transport: local-tools
```

Comment:
Does having `local-tools` before `test-transport` and `test-cli` mean that these get built twice? Or is `make` smart enough to know?